mdformat_admon/plugin.py

```diff
@@ -26,19 +26,22 @@ def _render_admon(node: RenderTreeNode, context: RenderContext) -> str:
     - https://github.com/executablebooks/mdformat-footnote/blob/80852fc20cfba7fd0330b9ac7a1a4df983542942/mdformat_footnote/plugin.py
 
     """
-    separator = "\n\n"  # TODO: Is this configurable?
-    indent = " " * 4  # FIXME: Is this configurable?
+    prefix = node.markup.split(" ")[0]
     title = node.info.strip()
-    title_line = f"!!! {title}"
+    title_line = f"{prefix} {title}"
+    # The indent is either 3 or 4 based on the length of the prefix
+    #   Ex: '!!!', '...', '..', '???', '???+', etc.
+    indent = " " * (min(len(prefix), 3) + 1)
```

Comment:
Per discussion in https://github.com/KyleKing/mdformat-admon/pull/4#discussion_r1049566735, only `!!!` is supported, but I plan to defer to `mdformat` for what constitutes an admonition block and attempt to enforce a deterministic style where I can

Reply:
Started in #7